src/video/sdl2_v.cpp

```diff
@@ -517,6 +519,10 @@ const char *VideoDriver_SDL_Base::Initialize()
 
 	const char *error = InitializeSDL();
 	if (error != nullptr) return error;
+	SDL_version linked;
+	SDL_GetVersion(&linked);
+	this->sdl_lib_major = linked.major;
+	this->sdl_lib_minor = linked.minor;
```

Comment:
Why not just store the `SDL_version` struct, rather than splitting it out like this?

Reply:
Maybe using `SDL_VERSION_ATLEAST` would drop the necessity to store version information altogether.